packages/audio-block/src/components/ItemToolbar.tsx

```diff
@@ -0,0 +1,135 @@
+                                                      },
+                                                      initialValue: true,
+                                                      decorator: (
+                                                          <div className="tw-mr-2">
+                                                              <IconArrowCircleUp20 />
+                                                          </div>
+                                                      ),
+                                                  },
+                                              ]
+                                            : []),
+                                        ...(!!onAssetChooseClick
+                                            ? [
+                                                  {
+                                                      id: 'asset',
+                                                      size: MenuItemContentSize.XSmall,
+                                                      title: 'Replace with asset',
+                                                      onClick: () => {
+                                                          setIsFlyoutOpen(false);
+                                                          onAssetChooseClick();
+                                                      },
+                                                      initialValue: true,
+                                                      decorator: (
+                                                          <div className="tw-mr-2">
+                                                              <IconImageStack20 />
+                                                          </div>
+                                                      ),
+                                                  },
+                                              ]
+                                            : []),
+                                    ],
+                                },
+                                {
+                                    id: 'deletemenu',
+                                    menuItems: [
+                                        {
+                                            id: 'delete',
+                                            size: MenuItemContentSize.XSmall,
+                                            title: 'Delete',
+                                            onClick: onRemoveAsset,
+                                            initialValue: true,
+                                            decorator: (
+                                                <div className="tw-mr-2">
+                                                    <IconTrashBin20 />
+                                                </div>
+                                            ),
+                                        },
+                                    ],
+                                },
+                            ]}
```

Comment:
Please move to a variable

Reply:
move into a var within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/97abd2a0e4237e89ae18e10d2cc7e4290bae98d1)